Sources/Logging/Logging.swift

```diff
@@ -531,7 +554,8 @@ public struct StdoutLogHandler: LogHandler {
         let prettyMetadata = metadata?.isEmpty ?? true
             ? self.prettyMetadata
             : self.prettify(self.metadata.merging(metadata!, uniquingKeysWith: { _, new in new }))
-        print("\(self.timestamp()) \(level):\(prettyMetadata.map { " \($0)" } ?? "") \(message)")
+        var output = FileOutputStream(file: stream == .stdout ? stdout : stderr)
```

Comment:
creating a new instance of FileOutputStream on every log call seems expensive? since stream is mutable, maybe move to ctor?

Reply:
I had the same though on this, but was following the setup from the other PR. I'll revisit

Reply:
yup. its used in as an inout param in L558 so needs to be a local var, but i am not sure if it'd be cheaper to create on the ctor and take a local copy or this is fine as is. cc @weissi maybe he has some ideas

Reply:
I changed it! Take a look

Reply:
My screen didn't refresh, just saw your latest comment. I have the same unsure-ness about the way I had it vs the way I changed it to regarding performance impact.

Reply:
@tomerd / @allenhumphreys `FileOutputStream` is a `struct` that holds only a pointer. Creating this there is just fine and free like putting an integer there :).

In other words, the code up there is exactly as expensive as

    var output = stream == .stdout ? 1 : 2